packages/terra-folder-tree/src/subcomponents/FolderTreeItem.jsx

```diff
@@ -122,6 +175,10 @@ const FolderTreeItem = ({
         aria-expanded={isFolder ? isExpanded : null}
         aria-selected={isSelected}
         onClick={isFolder ? handleToggle : onClick}
+        onKeyDown={handleKeyDown}
+        data-item-show-focus
```

Comment:
Just a question.  What is the purpose of this data attribute?

Reply:
This is used here https://github.com/cerner/terra-framework/pull/1956/files/ea0913c1bbd0b9a3e889d21308d922ff24573ae9#diff-622b2c4e339604395a6822a8aa1db5c37de2c6b45a8b6cd9878fb644c680ec1cR61

This gets all the list items that can be interacted with

Reply:
Should we remove the elements that can't be interacted with from the DOM?

Reply:
Continued discussion in https://github.com/cerner/terra-framework/pull/1956#discussion_r1430685528